diff -up dialog-1.1-20071028/makefile.in.libs dialog-1.1-20071028/makefile.in
--- dialog-1.1-20071028/makefile.in.libs	2007-07-04 21:33:35.000000000 +0200
+++ dialog-1.1-20071028/makefile.in	2007-10-30 17:17:46.000000000 +0100
@@ -122,11 +122,11 @@ dialog$o \
 $(OBJECTS) : $(srcdir)/dialog.h $(srcdir)/dlg_keys.h dlg_config.h VERSION
 
 $(LIB) : $(LIB_OBJECT)
-	$(LIBTOOL_CREATE) $(LIB) $(LIB_OBJECT)
+	$(LIBTOOL_CREATE) $(LIB) $(LIB_OBJECT) $(LIBS)
 	$(RANLIB) $@
 
 dialog$x : $(LIB) dialog$o @INTLDIR_MAKE@ @INTLLIBS@
-	$(LINK) -o $@ dialog$o -L. -ldialog $(LDFLAGS) $(LIBS)
+	$(LINK) -o $@ dialog$o -L. -ldialog $(LDFLAGS)
 
 clean	\
 distclean \
